man/pam_u2f.8.txt

```diff
@@ -162,6 +170,35 @@ mapping file in an encrypted home directory, will result in the
 impossibility of logging into the system. The partition is decrypted
 after login and the mapping file can not be accessed.
 
+== CONFIGURATION FILE
+A configuration file can be supplied in order to provide system-level default
+values for the `pam_u2f.so` module arguments.
+
+- The file can contain the same arguments as defined in the *OPTIONS* section,
+  one per line, exception made for the `conf_file=` argument, which can only
+  appear under `/etc/pam.d/`.
+
+- Lines starting with `#` are interpreted as comments and skipped, together with
+  empty lines.
+
+- In every line heading white-spaces (intended as in `isspace(3)`) are ignored.
+  Everything from the first non-white-space character till the end of line is
+  interpreted verbatim. Note that it is not possible to add trailing comments
+  to lines.
+
+- Square brackets around arguments, usually needed under `/etc/pam.d` to get
+  spaces correctly recognized, must not be supplied.
+
+The default path of the system-level configuration file is
+`/etc/security/pam_u2f.conf`.  The file is allowed to be missing or empty, in
```

Comment:
Do we need to take the distribution-configured value into account here?

In general, see notes from README.

Reply:
Yeah, the manpage follows the README for what I'm concerned, so I'll just apply changes to both.